.github/workflows/autofix.yml

```
@@ -36,7 +36,18 @@ jobs:
       - name: json formatting
         run: make style-all-json-parallel RELEASE=1
 
-      - uses: autofix-ci/action@8caa572fd27b0019a65e4c695447089c8d3138b9
+      - uses: autofix-ci/action@d3e591514b99d0fca6779455ff8338516663f7cc
         if: ${{ always() }}
         with:
           commit-message: "style(autofix.ci): automated formatting"
+          commment: |
+            The app has automatically formatted this PR.
+
+            If you edit your PR on web UI, you can ignore this message.
+            If you edit your PR locally, YOU MUST DO EITHER OF THE FOLLOWING:
+
+            - Run `git pull` to merge the automated commit into your PR branch.
+            - [Format your code locally](https://docs.cataclysmbn.org/en/mod/json/explanation/json_style/), and force push to your PR branch.
```

Autofix formats not only JSON
```suggestion
            - [Format your code locally](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style), and force push to your PR branch.
```